modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java

```diff
@@ -214,7 +217,11 @@ public void testSendApprovalPost() throws Exception {
 
 		accessToken = DataExtractUtil.extractParamFromURIFragment(locationHeader.getValue(),
 				OAuth2Constant.ACCESS_TOKEN);
+		String urlScopes = DataExtractUtil.extractParamFromURIFragment(locationHeader.getValue(),
+				OAuth2Constant.SCOPE_PLAYGROUND_NAME);
```

Comment:
Use the `OAuth2Constant.OAUTH2_SCOPE` constant instead

```suggestion
				OAuth2Constant.OAUTH2_SCOPE);
```

Reply:
Fixed